api/DOMMatrix.json

```diff
@@ -96,6 +96,288 @@
           }
         }
       },
+      "invertSelf": {
```

Comment:
Confirmed this and the other new entries here aren't already in some other part of BCD, none of the strings appear anywhere.